cds_ils/cli.py

```diff
@@ -66,11 +66,39 @@ def pages():
 def location():
     """Create library location."""
     click.echo("Creating locations...")
+    weekdays = [
+        "monday",
+        "tuesday",
+        "wednesday",
+        "thursday",
+        "friday",
+        "saturday",
+        "sunday",
+    ]
+
+    closed = ["saturday", "sunday"]
+    times = [
+        {"start_time": "08:00", "end_time": "12:00"},
+        {"start_time": "13:00", "end_time": "18:00"},
+    ]
+    opening_weekdays = []
```

Comment:
nit: I noticed an inconsistency in the tense of the verbs `closed` past, vs `opening` indicates duration etc, maybe `open` and `open_weedkdays` is more "correct"

Reply:
I see its comming from ILS you can ignore my comment.